src/Fabulous.XamarinForms/Views/GestureRecognizers/DropGestureRecognizer.fs

```diff
@@ -0,0 +1,68 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Xamarin.Forms
+
+type IDropGestureRecognizer =
+    inherit Fabulous.XamarinForms.IGestureRecognizer
+
+module DropGestureRecognizer =
+    let WidgetKey =
+        Widgets.register<DropGestureRecognizer> ()
+
+    let AllowDrop =
+        Attributes.define<bool>
```

Comment:
I see they have a bindable property for this one: https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.dropgesturerecognizer.allowdropproperty?view=xamarin-forms

Reply:
Yeah I missed it 😀